Dockerfile-csi-test

```diff
@@ -15,6 +15,7 @@
 FROM golang:1.11
 
 ARG CSI_PARAMS=csi_params
+ARG STORAGE_LABEL=community_svc
```

Comment:
Why do we use SCV in this case? Do we have similar labels for a9k/ds8k?

Reply:
it's just a default value. 
we have label for each storage and we pass the relevant label through the docker build command